Sources/Public/BacktraceCrashReporter.swift

```diff
@@ -48,7 +53,11 @@ extension BacktraceCrashReporter: CrashReporting {
                             attributes: Attributes,
                             attachmentPaths: [String] = []) throws -> BacktraceReport {
 
+#if SWIFT_PACKAGE
```

Comment:
Why is there an ifdef here, why does it need to be different for SWIFT_PACKAGE vs other build options?

Reply:
With Swift package manager we are using https://github.com/microsoft/plcrashreporter.git which is orignal branch with latest code but with framework we are still using Backtrash_CrashReporter which is significantly old one. so as per the build oprion we picked implementation.

Reply:
This should be removed as per the previous comment,